cmd/root_environment.go

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2017 The Kubicorn Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/viper"
+)
+
+const (
```

Comment:
This is also a thing that we don't addressed now, but that could make a good discussion and become a good issue to work on.

Can we make a map of this variables instead of constants?
I believe constants could be better in cases of memory management, but if we could make a map variant, that could be better in terms of code readability (and we wouldn't need to repeat some logic).